admin/src/scenes/volontaires/view/phase2MilitaryPreparation.js

```diff
@@ -100,7 +100,7 @@ export default function Phase2militaryPrepartion({ young }) {
     console.log("onRefuse");
 
     // update the young
-    const responseYoung = await api.put(`/referent/young/${young._id}`, { statusMilitaryPreparationFiles: "REFUSED" });
+    const responseYoung = await api.put(`/referent/young/${young._id}/refuse-military-preparation-files`, { statusMilitaryPreparationFiles: "REFUSED" });
```

Comment:
pourquoi pas faire ca ducoup ?
```suggestion
    const responseYoung = await api.post(`/referent/young/${young._id}/refuse-military-preparation-files`);
```